frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -66,36 +91,47 @@ export class TemplateValidations {
 
     // Check if two sets of bus validations are the same - if the allowed and recommended buses are the same
     const allowedBuses = this.getAllowedBuses();
+    const allowedCDBuses = this.getAllowedCDBuses();
     const recommendedBuses = this.getRecommendedBuses();
+    const recommendedCDBuses = this.getRecommendedCDBuses();
     const otherAllowedBuses = otherTempValidations.getAllowedBuses();
+    const otherAllowedCDBuses = otherTempValidations.getAllowedCDBuses();
     const otherRecommendedBuses = otherTempValidations.getRecommendedBuses();
+    const otherRecommendedCDBuses = otherTempValidations.getRecommendedCDBuses();
 
     return (
```

Comment:
maybe we could do a lazy compare here? First get AllowedBuses and compare if not equal return. Then get aand compare recommended. Etc.